pypeman/commands.py

```diff
@@ -88,15 +137,20 @@ def main(debug_asyncio=False, profile=False, cli=False, remote_admin=False):
 
     # At the moment mixing the asyncio and ipython
     # event loop is not working properly.
-    # Thus ipython
+    # Thus ipython is executed in a separate thread
     if cli:
         from pypeman.helpers.cli import CLI
         namespace = dict(
+            fake_signal_int=partial(signal_handler, signal.SIGINT, None),
```

Comment:
What does this parameter do ?

Reply:
this is for simulating the reception of an INT signal (stopping of all loops) from within the CLI.